app/services/sync_provider_from_find.rb

```diff
@@ -82,7 +86,7 @@ def self.create_or_update_course(find_course, provider)
           site: site,
```

Comment:
I think we need to update the logic here to update rather than `find_or_create_by`, otherwise existing courses won't be updated.

Reply:
Good point, updating...